src/gosling-genomic-axis/axis-track.ts

```diff
@@ -20,12 +20,19 @@ type TickLabelInfo = {
     rope: PIXI.SimpleRope;
```

Comment:
Related to my comment below.

Reply:
So the type would need to be:

```typescript
    rope?: PIXI.SimpleRope | null;
```

 to allow this code to type-check completely. Allowing a field to be `null` | `undefined` is a pattern I really try to avoid since it leads to complicated interfaces like this, and semantically `null` and `undefined` usually mean the same thing. My personal preference is to avoid `null` in JavaScript, since we have also have `undefined` (nice post here on topic https://github.com/sindresorhus/meta/discussions/7), but I'm fine with choosing one or the other.